src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
-body {
-  margin: 0;
+$gap-size: 1px;
+$cells-in-row: 7;
+$height-cell: 100px;
+$width-cell: 100px;
+$week-days:
+  'mon' 1,
+  'tue' 2,
+  'wed' 3,
+  'thu' 4,
+  'fri' 5,
+  'sat' 6,
+  'sun' 7;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $width-cell * $cells-in-row + $gap-size * ($cells-in-row - 1);
```

Comment:
The calculation here is not following the DRY principle. You're repeating `$width-cell * $cells-in-row` and `$gap-size * ($cells-in-row - 1)`. Consider creating a variable for `$width-cell * $cells-in-row` as it represents the total width of the cells without gaps and reuse it for the width calculation.